bats_test/ardrive.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+
+@test "'ardrive' with no input arguments prints general help with all expected commands" {
+    run yarn ardrive
+    [ "${status}" -eq 0 ]
+    [[ "$output" = "Usage: ardrive [command] [command-specific options]
```

Comment:
I'd rather declare the multi-line string in a separate variable so the conditional gets more clear. That way we can use the single brackets instead of double brackets (I think the multi-line string is the reason we are using `[[`);

Also, a minor detail:
```suggestion
    [[ "${output}" = "Usage: ardrive [command] [command-specific options]
```

Reply:
Fixed that test and added some more